content/docs/details/decorating-components/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Catalog is built around the concept of [entity descriptor YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/), therefore, any change you wish to make to your components, needs to be done via editing those files. 
```

Comment:
This isn't quite true. The catalog doesn't depend on YAML files but can be populated using any methods